module/VuFind/src/VuFind/Controller/Search2recordController.php

```diff
@@ -65,4 +65,14 @@ protected function resultScrollerActive()
         return isset($config->Record->next_prev_navigation)
             && $config->Record->next_prev_navigation;
     }
+
+    /**
+     * Get the Hierarchy Type (false if none)
+     *
+     * @return string|bool
+     */
+    public function getHierarchyType()
```

Comment:
Does this method do anything? I'm not seeing any getHierarchyType method used/defined in the core code's controllers -- this seems to be a record driver mechanism.

Reply:
You're right, it's not needed here.